crates/wash-cli/src/app/mod.rs

```diff
@@ -216,10 +216,35 @@ async fn deploy_model(cmd: DeployCommand) -> Result<DeployModelResponse> {
         None => load_app_manifest("-".parse()?).await?,
     };
 
-    match app_manifest {
+    deploy_model_from_manifest(&client, lattice, app_manifest, cmd.version).await
+}
+
+pub(crate) fn get_model_name(manifest: &AppManifest) -> Result<&str> {
```

Comment:
Nit: Should this just be a method on `AppManifest`?

Reply:
Sure, a follow up PR to `wadm` to add it would be great -- tracking [that here](https://github.com/wasmCloud/wadm/issues/293)